generate-sitemap.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const path = require('path');
+
```

Comment:
Can you move this file (and the blacklist file) to the scripts folder?

Also, can you rename the blacklists file to not be prefixed by a dot? Leaving it with a dot makes it not show up automatically as a changed file in version control.

Reply:
@kylemh  All necessary changes are done.